auth/pom.xml

```diff
@@ -104,11 +108,6 @@ limitations under the License.
         </executions>
         <configuration>
           <executable>java</executable>
-          <arguments>
-            <argument>-classpath</argument>
-            <classpath />
```

Comment:
please note I had to remove this specification as it was causing all AuthExamples tests to fail.  I have updated the readme so DownscopingExample can run without it.